Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RPISystemInterface.h

```diff
@@ -90,6 +90,10 @@ namespace AZ
 
             //! Get the index of current render tick
             virtual uint64_t GetCurrentTick() const = 0;
+
+            //! Application multisample state
+            virtual void SetApplicationMultisampleState(const RHI::MultisampleState& multisampleState) = 0;
```

Comment:
Minor nit: it won't be obvious to a user/reader not familiar with the rendering pipeline which render targets specifically might be affected by this setting, so a comment to the effect of how this setting is used (or a pointer to where in the pass setup this is used) would be useful.